man/docker-commit.1.md

```diff
@@ -17,6 +17,7 @@ CONTAINER [REPOSITORY[:TAG]]
 Create a new image from an existing container specified by name or
 container ID.  The new image will contain the contents of the
 container filesystem, *excluding* any data volumes.
+[Read more about valid image names and tags](https://docs.docker.com/reference/commandline/tag/).
```

Comment:
I'm not sure these links will show up once these files are postprocessed to create manual pages. Does anyone know how this is handled?